extensions/BMO/template/en/default/bug/create/create-ipc.html.tmpl

```diff
@@ -167,7 +167,7 @@ $(document).ready(function() {
     </div>
 
     <p>
-    If you would like your snippet to include a link, please tag it by using the URL builder <a href="http://bit.ly/1JOgDr1">(http://bit.ly/1JOgDr1)</a> and following the steps below:
+    If you would like your snippet to include a link, please tag it by using the URL builder <a href="https://bit.ly/1JOgDr1">(https://bit.ly/1JOgDr1)</a> and following the steps below:
```

Comment:
I don't like these bit.ly links. We likely have no reason to use a URL shortener here, and if we do, we need to use mzl.la for it.

Reply:
I don't know why those bit.ly links exist, but it's probably out of scope.